src/modules/core/components/SingleUserPicker/SingleUserPicker.tsx

```diff
@@ -233,7 +234,8 @@ const SingleUserPicker = ({
                   {value.profile.displayName ||
                     value.profile.username ||
                     value.profile.walletAddress}
-                  {showMaskedAddress && (
+                  {/* eslint-disable-next-line max-len */}
```

Comment:
I had to add this line as there seemed to be a weird bug with stylelint and eslint. I can't add a line break because it gets "fixed" back to one line, presumably by stylelint, and then eslint flags this as an issue. Have you come across this before?

Reply:
I've come across this but with strings, but in this case, I don't think there's a solution other than storing the wallet address in a const so that you can do `... isAddress(userAddress)` or something like that